eyelid-match-ops/src/encoded/conf.rs

```diff
@@ -59,53 +67,63 @@ pub trait EncodeConf {
     }
 }
 
-// TODO: add a conf where EyeConf and PlainConf are different, and test it.
-
-impl EncodeConf for IrisBits {
-    type EyeConf = IrisBits;
-    type PlainConf = IrisBits;
+impl EncodeConf for FullBits {
+    type EyeConf = FullBits;
+    type PlainConf = FullRes;
 
-    const ROWS_PER_BLOCK: usize = 20;
+    // TODO: using 16 fails with an encryption positive multiplication test error
+    const ROWS_PER_BLOCK: usize = 8;
 }
+// As in the report
+const_assert_eq!(
+    <<FullBits as EncodeConf>::PlainConf as PolyConf>::MAX_POLY_DEGREE,
+    2048
+);
```

Comment:
@bgillesp full resolution parameters: rows per block, and polynomial degree for full resolution blocks

Reply:
Confirmed, 8 rows per block for degree 2047 polynomials should be fine for the 200-bit rows with rotation amounts -15 to +15.